src/software/thunderscope/field/passing_layer.py

```diff
@@ -73,35 +58,30 @@ def paint(self, painter, option, widget):
             pass_vis.best_passes, key=lambda x: x.rating, reverse=True
         )
 
-        for pass_num, pass_with_rating in enumerate(
-            sorted_pass_with_rating[0 : PassingLayer.NUM_TOP_PASSES_TO_RENDER]
-        ):
-
-            # Lets color the best pass green and rest white
-            painter.setPen(
-                pg.mkPen(
-                    "green" if pass_num == 0 else "white", width=constants.LINE_WIDTH
-                )
-            )
-
-            painter.drawEllipse(
-                self.createCircle(
-                    pass_with_rating.pass_.receiver_point.x_meters
-                    * MILLIMETERS_PER_METER,
-                    pass_with_rating.pass_.receiver_point.y_meters
-                    * MILLIMETERS_PER_METER,
-                    ROBOT_MAX_RADIUS_MILLIMETERS * 4,
-                )
-            )
-
-            # Draw the pass rating
-            text_item = self.pass_rating_text_items[pass_num]
-            text_item.setText(
-                str("Pass {} {:.2f}".format(pass_num, pass_with_rating.rating))
-            )
-            text_item.setPos(
-                pass_with_rating.pass_.receiver_point.x_meters * MILLIMETERS_PER_METER
-                - 4 * ROBOT_MAX_RADIUS_MILLIMETERS,
-                pass_with_rating.pass_.receiver_point.y_meters * MILLIMETERS_PER_METER
-                - 4 * ROBOT_MAX_RADIUS_MILLIMETERS,
-            )
+        painter.setPen(pg.mkPen("red", width=2, style=QtCore.Qt.PenStyle.DotLine))
```

Comment:
I personally find the red line kind of distracting, as it often goes across the field and is constantly moving. I think red with lower opacity (like `QtGui.QColor(255, 0, 0, 80)`) looks better and less distracting.
I also wonder how hard it is to make the dotted (or dashed) line have its dots further apart. Right now on my computer the line looks pretty solid given how close the dots are.

Reply:
i like the dash and prefer it compared to the line